src/lib/openapi/spec/event-schema.ts

```diff
@@ -1,51 +1,136 @@
 import { FromSchema } from 'json-schema-to-ts';
 import { tagSchema } from './tag-schema';
+import { IEventTypes } from '../../types';
+import { variantSchema } from './variant-schema';
 
+const eventDataSchema = {
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        name: {
+            type: 'string',
+        },
+        description: {
+            type: 'string',
+        },
+        type: {
+            type: 'string',
+        },
+        project: {
+            type: 'string',
+        },
+        stale: {
+            type: 'boolean',
+        },
+        variants: {
+            type: 'array',
+            items: {
+                $ref: '#/components/schemas/variantSchema',
+            },
+        },
```

Comment:
I know these properties are only here to allow the example (or at least that's what I seem to remember), but could we still give them a description? It might be hard because they can vary between events, though 🤔

Reply:
Well, I can still give it a go certainly, I can steal with pride from our feature schema :+1:

Reply:
Sounds good! Just try and make it clear in some way that the descriptions may apply to different things depending on what the event is for